src/routes/docs/products/messaging/targets/+page.markdoc

```diff
@@ -4,4 +4,20 @@ title: Targets
 description: Manage avenues of communication by targetting user's device, email, or phone number in your notification and messages.
 ---
 
+Targets are different ways a user can be reached.
+For example, a user might have two emails, a phone number as well as a phone and a tablet with your app installed.
+This means, the user has five different targets that you can deliver messages to.
+
+# Topics and targets {% #topics-and-targets %}
+A user can have multiple targets, such as emails, phone numbers, and devices with your app installed. 
+These targets can subscribe to a topic, so when messages are published to a topic, all subscribed targets receive the message.
+
+{% arrow_link href="/docs/products/messaging/topics" %}
+Learn more about topics
+{% /arrow_link %}
+
+# Types of targets {% #types-of-targets %}
+
+[@abnegate @stnguyen90: TODO types of targets and how to create/add targets in console + code.]
```

Comment:
@abnegate @stnguyen90 What are the different target types called?

What's the code like for creating targets?

Is there client code required?

Reply:
> What are the different target types called?

```php
// Message types
const MESSAGE_TYPE_EMAIL = 'email';
const MESSAGE_TYPE_SMS = 'sms';
const MESSAGE_TYPE_PUSH = 'push';
```

https://github.com/appwrite/appwrite/blob/49a753509b17ee4209999e08678e8cad71a20bee/app/init.php#L197-L199

> What's the code like for creating targets?

[Client-side](https://github.com/appwrite/appwrite/blob/49a753509b17ee4209999e08678e8cad71a20bee/app/controllers/api/account.php#L1951-L1972), you can only create a push target:

```
account.createPushTarget()
```

[Server-side](https://github.com/appwrite/appwrite/blob/49a753509b17ee4209999e08678e8cad71a20bee/app/controllers/api/users.php#L429-L452), you can create any target:

```
users.createTarget()
```

> Is there client code required?

Did I answer your question?